DATA_DICTIONARY.md

```diff
@@ -0,0 +1,71 @@
+# Data Dictionary for Super DCA Metrics
+
```

Comment:
Include a "Metrics" table which works like a glossary, two columns: Metric, Description.

Include a formula when the metric is more than a simple average or sum, e.g. for the "Oracle Delta" or whatever we want to call it.